internal/watch/watch.go

```diff
@@ -288,3 +289,81 @@ func renderLogs(ctx context.Context, alloc *api.AllocationStatus) {
 		)
 	}
 }
+
+func MachineChecks(ctx context.Context) error {
+	var (
+		io          = iostreams.FromContext(ctx)
+		flapsClient = flaps.FromContext(ctx)
+	)
+
+	tb := render.NewTextBlock(ctx, "Monitoring health checks")
+
+	tb.Println()
+
+	var pass, warn, fail int
+
+	var errCount int
+
+	ctx, cancel := context.WithTimeout(ctx, 120*time.Second)
```

Comment:
We should avoid using a global timeout here.  What's reasonable will be based on the number of machines we are monitoring.